### PR TITLE
Move Sequence to Task RecoJets RecoMET

### PR DESCRIPTION
**Pull request by jeongeun**
#### PR description:
Move Sequence to Task for RecoJets RecoMET cff python files.
(An extension of the previous task [PR#25163](https://github.com/cms-sw/cmssw/pull/25163), [PR#27474](https://github.com/cms-sw/cmssw/pull/27474) , [PR#27640](https://github.com/cms-sw/cmssw/pull/27640))

Tested in CMSSW_11_0_X_2019-08-26-1100 release, The PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html).

### CODE REVIEW

**Comment by jeongeun** [comment]
please test

**Comment by jeongeun** [comment]
[@cmsbuild,] please test

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- The replacement of sequences with tasks is implemented in line with the other PRs in this campaign
- Jenkins tests pass and show no differences

**Comment by santocch** [comment]
+1

**Comment by kpedro88** [comment]
+1

File: JetMETCorrections/Configuration/python/JetCorrectorsForReco_cff.py [hunk 1]
```diff
@@ -6,6 +6,5 @@
 from JetMETCorrections.Configuration.JetCorrectors_cff import ak4PFCHSL1FastL2L3CorrectorChain,ak4PFCHSL1FastL2L3CorrectorTask
```
Comment (jeongeun) [inline JetMETCorrections/Configuration/python/JetCorrectorsForReco_cff.py]: Sorry for the late response to this. 
I missed changing ak4PFCHSL1FastL2L3Corrector**Chain** to ak4PFCHSL1FastL2L3Corrector**Task**.

I will fix it.

**Pull request merged**